api/v1alpha1/componentversion_types.go

```diff
@@ -49,9 +49,16 @@ type ComponentVersionSpec struct {
 	// +required
 	Interval metav1.Duration `json:"interval"`
 
+	// Every Component Version has a name.
+	// Name and version are the identifier for a Component Version and therefor for the artifact set described by it.
+	// A component name SHOULD reference a location where the component’s resources (typically source code, and/or documentation) are hosted.
+	// Examples:
+	// - github.com
```

Comment:
According to the ocm-spec, `github.com` is not a valid component name. :) It MUST contain a name after the domain. So, `github.com/<name>` is valid. And it MUST be a DNS compliant name with lowercase characters.

Reply:
Heh, that's funny. Than the official spec is wrong. :)